CompilerSource/JDI/src/API/AST_Export.cpp

```diff
@@ -22,6 +22,8 @@
 #include "AST.h"
 #include <General/svg_simple.h>
 
+#include <algorithm> // std::max
```

Comment:
`<algorithm>` is a C++ header and should go above a C header like `<svg_simple.h>`

Reply:
nope it should go at the bottom but includes in that file are backwards